src/langcheck/metrics/prompts/en/confidence_estimating/simulated_annotators.j2

```diff
@@ -0,0 +1,18 @@
+You are a helpful assistant.
+
+Given a question and two assistant's answers to the question, an annotator chose which assistant's answer is preferred.
+Given examples of the annotator's decision, predict the annotator's verdict on the given example.
+If Assistant A's response is preferred to Assistant B's, the annotator chose “A”.
+If Assistant B's response is preferred to Assistant A's, the annotator chose “B”.
+
+[Few-shot examples]
```

Comment:
I think we could add {{ few_shot_examples }} here too and then inject them in here instead of using split